src/components/Sidebar.jsx

```diff
@@ -0,0 +1,30 @@
+import { BsFillTrashFill } from "react-icons/bs";
+
+export default function Sidebar(props) {
+
+    const noteElements = props.notes.map((note, index) => (
+        <div key={note.id}>
+            <div className={`title ${note.id === props.currentNote.id ? "selected-note" : ""}`}
```

Comment:
since you only have one alternative, you can make this a logical AND operator 
```suggestion
            <div className={`title ${note.id === props.currentNote.id && "selected-note"}`}
```